apps/backend/src/routes/habit.ts

```diff
@@ -136,4 +136,45 @@ router.delete(
   })
 );
 
+// Edit a habit
+router.put(
+  "/:id",
+  asyncHandler(async (req, res) => {
+    const habitId = parseInt(req.params.id);
+    const { name, description, endDate } = req.body as {
```

Comment:
`description` is unused

Reply:
This is because we currently do not store any description data on the habits. If this ever gets added to the Prisma models then we can utilize the update description.